charts/fulcio/templates/fulcio-deployment.yaml

```diff
@@ -59,14 +59,27 @@ spec:
             - "--kms-cert-chain-path=/etc/fulcio-config/chain.pem"
             {{- end }}
             - "--ct-log-url={{ if .Values.server.args.disable_ct_log }}{{ else if .Values.server.args.ct_log_url }}{{ .Values.server.args.ct_log_url }}{{ else }}http://{{ .Values.ctlog.name }}.{{ .Values.ctlog.namespace.name }}.svc/{{ .Values.ctlog.createctconfig.logPrefix }}{{ end }}"
-          {{- if eq .Values.server.args.certificateAuthority "fileca" }}
+{{- if .Values.server.env }}
```

Comment:
This doesnt have a closing end tag, yet strangely no error is thrown

Reply:
The closing `{{- end}}` for line 64 is at [line82](https://github.com/sigstore/helm-charts/pull/530/files#diff-67bab0f130b70c4c9610b3450c237a502b48b40e743565e4806a71a6e3f65c11R82)

Reply:
Got it

To simplify, this condition should be an `or` statement that checks the presence of `Values.server.env` or whether `Values.server.args.certificateAuthority` == `fileca`. This would avoid the redundant `PASSWORD` env variable definition

Reply:
that is still fine. having the proposed conditional would still suffice. [this conditional](https://github.com/sigstore/helm-charts/blob/10128339c6ebe312eb4271ad678ffa58a2d28215/charts/fulcio/templates/fulcio-deployment.yaml#L68) would remain in order to capture when it was defined, otherwise only the key/values would be captured. The contents within [this conditional](https://github.com/sigstore/helm-charts/blob/10128339c6ebe312eb4271ad678ffa58a2d28215/charts/fulcio/templates/fulcio-deployment.yaml#L75) block can be removed as its no longer needed